cmd/draft/draft.go

```diff
@@ -108,13 +108,13 @@ func teardown() {
 	}
 }
 
-func ensureDraftClient(p *draft.Client) *draft.Client {
-	if p != nil {
-		return p
-	}
-	client, err := draft.NewFromString(draftHost, nil)
-	if err != nil {
-		panic(err)
+func ensureDraftClient(client *draft.Client) *draft.Client {
+	if client == nil {
+		return draft.NewClient(&draft.ClientConfig{
+			ServerAddr: draftHost,
+			Stdout:     os.Stdout,
+			Stderr:     os.Stderr,
```

Comment:
we also might want to add a `Stdin` field here as well, as `draft init` and some of the Draft plugins read from stdin.